iotlabcli/parser/common.py

```diff
@@ -128,6 +128,10 @@ def catch_missing_auth_cli():
 def main_cli(function, parser, args=None):  # flake8: noqa
     """ Main command-line execution. """
     args = args or sys.argv[1:]
+    if len(args) == 0:
```

Comment:
Maybe a test could be added somewhere [around there](https://github.com/iot-lab/cli-tools/blob/master/iotlabcli/tests/common_parser_test.py#L37)